terraform/github/repositories/imports.tf

```diff
@@ -0,0 +1,131 @@
+# Imports to ministryofjustice-test org team_id 9472191 with push access
+import {
+  to = module.test_tamf_repo_test_org.github_team_repository.push["9472191"]
+  id = "9472191:test-tamf-repo-test-org"
+}
+
+# Imports to ministryofjustice operations-engineering team_id 4192115 with admin access
+import {
+  to = module.acronyms.github_team_repository.admin["4192115"]
+  id = "4192115:acronyms"
```

Comment:
Isn't the organization supposed to be referenced in the id of the resource being imported and not the repository?

Reply:
So the organization is collected via provider, which is referenced in the module and in the data source that collects the team name, unless it is the default org (ministryofjustice). I can only presume that the module provider block used when the ministryofjustice-test org is required is sufficient to ensure that the `id` is looked for in the correct org. You can't create associations between teams and repos unless they are in the same org.